src/graphql/schema/uuid/abstract-uuid/resolvers.ts

```diff
@@ -53,6 +53,17 @@ export const resolvers: AbstractUuidResolvers = {
             return uuid && uuid.__typename === 'User' ? uuid : null
           }
         }
+
+        //TODO: to support legacy feature. remove after frontend is 100% (#143)
+        if (cleanPath.startsWith('/entity/view/')) {
+          const match = /^\/entity\/view\/(\d+)$/.exec(cleanPath)
+          if (match) {
+            const id = parseInt(match[1], 10)
+            const uuid = await dataSources.serlo.getUuid<UuidPayload>({ id })
+            return uuid || null
+          }
+        }
```

Comment:
```suggestion
          const match = /^\/entity\/view\/(\d+)$/.exec(cleanPath)
          if (match) {
            const id = parseInt(match[1], 10)
            return await dataSources.serlo.getUuid<UuidPayload>({ id })
          }
        
```

First `if` statemenent is redundant

Reply:
just went along with the profile example above, thought it's some tiny performance gain :)
Will change it above then as well, right?